index.html

```diff
@@ -0,0 +1,141 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <div class="logo">My Logo</div>
+      <div class="hum-menu">
+        <svg
+          width="40"
+          height="40"
```

Comment:
- [ ] It would be wise to use a `<nav>` tag instead of using a `<div>` one for your menu button. 💯 This is because the `<nav>` tag is recognized by the browser and screen readers as the element that's in charge of the navigation of your page. ⛵ Getting, as a result, better semantics and also a better experience for those users who rely on screen readers to navigate through your page, making it more accessible. 🤓

Reply:
Thankyou @anagudelogu. I now understand why it would be wise to use <nav> instead of a div for the menu button. Based on your suggestion I now used the <nav> tag.